modules/osdk-golang-implement-controller.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-implement-controller_{context}"]
+= Implementing the controller
+
+After creating a new API and controller, you can implement the controller logic.
+
+.Procedure
+
+* For this example, replace the generated controller file `controllers/memcached_controller.go` with the `memcached_controller.go` link:https://github.com/operator-framework/operator-sdk/blob/v1.2.0/testdata/go/memcached-operator/controllers/memcached_controller.go[example implementation].
++
+The example controller executes the following reconciliation logic for each `Memcached` custom resource (CR):
```

Comment:
> ! execute verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.